CMakeLists.txt

```diff
@@ -85,28 +89,11 @@ if (NOT NO_SDL2)
 
 	if (SDL2_FOUND)
 		message(STATUS "SDL2 found! Enabling UI components.")
-		include_directories(${SDL2_INCLUDE_DIR})
```

Comment:
At least on my system, with SDL2 installed in `/usr/local/lib`, it fails to link SDL2 when building:

```
ld: library not found for -lSDL2
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [bin/c-ray] Error 1
make[1]: *** [CMakeFiles/c-ray.dir/all] Error 2
make: *** [all] Error 2
```